packages/kit/test/apps/basics/test/test.js

```diff
@@ -1153,6 +1153,61 @@ test.describe('Actions', () => {
 		await expect(page.locator('pre.formdata2')).toHaveText(JSON.stringify({ message: 'hello' }));
 	});
 
+	test('use:enhance with `enctype="application/x-www-form-urlencoded"` attribute in `<form>`', async ({
+		page
+	}) => {
+		await page.goto('/actions/enhance');
+
+		expect(await page.textContent('pre.formdata1')).toBe(JSON.stringify(null));
+		expect(await page.textContent('pre.formdata2')).toBe(JSON.stringify(null));
+
+		await page.locator('input[name="username_2"]').fill('foo');
+		await page.locator('button.form-enctype-submit').click();
+
+		await expect(page.locator('pre.formdata1')).toHaveText(JSON.stringify({ result: 'foo' }));
+		await expect(page.locator('pre.formdata2')).toHaveText(JSON.stringify({ result: 'foo' }));
+		await expect(page.locator('input[name="username_2"]')).toHaveValue('');
+	});
```

Comment:
We should try to test for the expected content-type request header instead. Maybe we can return the `Content-Type` request header as a the form action result and check if it is `application/x-www-form-urlencoded` by default?

Reply:
Might be worth reusing the tests from https://github.com/sveltejs/kit/pull/7195/files now that we're resolving the content-type header automatically, whereas the tests from that PR try to resolve it manually.

Reply:
@eltigerchino this has been addressed in ba87fe949f4c076565f439de8b74bf21aab0c0c0